src/metabase/search/config.clj

```diff
@@ -195,25 +205,11 @@
   [_]
   [:name])
 
-(defmethod searchable-columns-for-model "action"
-  [_]
-  [:name
-   :dataset_query
-   :description])
-
-(defmethod searchable-columns-for-model "card"
+(doseq [model ["card" "dataset" "action" "metric"]]
+ (defmethod searchable-columns-for-model model
```

Comment:
Those loops look worse to me than previous code. I think making it more dense (lesser amount of newlines) is fine, but searchability of code with defmethod in a loop is not great.

Reply:
Good point, I hadn't considered the benefit of being able to search for a given model. Updated 
80f42c7599cd60976808744a30fe4b5cc72a8236

Reply:
(this code is going very soon anyway, but its still nice to refactor easier-to-read code)